image_width: 1280
image_height: 720
camera_name: fake_cam
camera_matrix:
  rows: 3
  cols: 3
  data: [1000.0, 0.0, 640.0,
         0.0, 1000.0, 360.0,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1,0,0, 0,1,0, 0,0,1]
projection_matrix:
  rows: 3
  cols: 4
  data: [1000.0,0.0,640.0,0.0,
         0.0,1000.0,360.0,0.0,
         0.0,0.0,1.0,0.0]
